tests/infra_test.go

```diff
@@ -392,6 +394,10 @@ var _ = Describe("[Serial][sig-compute]Infrastructure", func() {
 
 		})
 
+		AfterEach(func() {
```

Comment:
There is a global AfterEach which should take care of that:

https://github.com/kubevirt/kubevirt/blob/main/tests/tests_suite_test.go#L100

and

https://github.com/kubevirt/kubevirt/blob/0f2c6714ae95371f14b34361f747c2311df18a4d/tests/utils.go#L134

Reply:
Yup. thx